### ISSUE TITLE
Rename move.NewMove to move.New()

### ISSUE THREAD

**Issue by artasparks** [otrego/clamshell#172]
Right now, you create new moves with the following:

```go
// NewMove creates a new Move.
func NewMove(col color.Color, pt *point.Point) *Move {
	return &Move{color: col, point: pt}
}
```

It would be simpler to have:

```go
// New creates a new Move.
func New(col color.Color, pt *point.Point) *Move {
	return &Move{color: col, point: pt}
}
```

Since you would invoke this is `move.New(col, pt)`, which reduces stutter.

Requirements:
- Rename NewMove to New.
- Fix any tests/docs along the way.

**Comment by michaelhiiva** [comment]
@Kashomon This issue seems to be resolved with #215.

**Comment by artasparks** [comment]
Oops! Yes.
